mobile/app/src/main/java/com/smartjam/app/data/api/InstantAdapter.kt

```
@@ -0,0 +1,31 @@
+        }
+    }
+
+    override fun read(`in`: JsonReader): Instant? {
+        if (`in`.peek() == JsonToken.NULL) {
+            `in`.nextNull()
+            return null
+        }
+        val s = `in`.nextString()
+        return try {
+            Instant.parse(s)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)